framework/src/components/cache/defaults/config.js

```diff
@@ -3,37 +3,39 @@ const defaultConfig = {
 	properties: {
 		enabled: {
 			type: 'boolean',
-			default: false,
 			env: 'LISK_CACHE_ENABLED',
 		},
 		host: {
 			type: 'string',
 			format: 'ipOrFQDN',
-			default: '127.0.0.1',
 			env: 'LISK_REDIS_HOST',
 			arg: '-r,--redis',
 		},
 		port: {
 			type: 'integer',
 			minimum: 1,
 			maximum: 65535,
-			default: 6380,
 			env: 'LISK_REDIS_PORT',
 		},
 		db: {
 			type: 'integer',
 			minimum: 0,
 			maximum: 15,
-			default: 0,
 			env: 'LISK_REDIS_DB_NAME',
 		},
 		password: {
 			type: ['string', 'null'],
-			default: null,
 			env: 'LISK_REDIS_DB_PASSWORD',
 		},
 	},
 	required: ['enabled', 'host', 'port', 'db', 'password'],
+	default: {
+		enabled: true,
```

Comment:
This was `false` before (cache enabled)?
Did we also not discuss to change `LISK_CACHE_ENABLED` to `LISK_ENABLE_CACHE` as it seems like a state instead a switch we can turn on/off?

Reply:
@shuse2 If we want to change variable names this release is the right candidate for it. The change on our side is very easy, need to communicate with devops team if we do it.

Reply:
yes we should make it clean! =)

Reply:
@michielmulders Please create a separate issue for refactoring the variable name. Mention this variable in the issue and then we ask everyone else if they want to suggest another improvement.